### DISCUSSION TITLE
Frequent gameplay freezes after this past update.

### DISCUSSION THREAD

**Discussion by mysteGH** [ppy/osu#26114] [Q&A]
### Type

Performance

### Bug description

Hey :D , logged into lazer today & updated my game, i've been playing on lazer for about 5 months now and have never had any performance issues at all but today i've had pretty long / frequent freezes (5 seconds+)
Sorry for no video, i had my replay overlay off (will attach video once i encounter it again)
I've exported my logs and below is a list of hardware i use. 

GPU: NVIDIA GTX 1070 (no OC) (driver ver 546.33)
CPU: AMD Ryzen 5 5600x (just game mode, no OC)
RAM: Corsair 16GB (8x2) 3200Mhz
Install Location: SSD (500GB, 100GB free)

Game running on:
 OpenGL (legacy), 4x Refresh Rate  (280hz context), Fullscreen exclusive. 

### Version

2023.1224.0

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/13760964/compressed-logs.zip)

**Comment by mysteGH** [comment]
https://github.com/ppy/osu/assets/96603029/6d898e38-dccc-4610-8670-82480beb3990

Another issue appears!: 
this to me screams gpu driver / hardware issue, but it's only present when playing osu, simply closing the game seemingly gets rid of whatever artefacting is happening here, if this is irreplicable and the issue falls on my pc i will close this thread, thank you guys for looking into this in advance though.

**Comment by peppy** [comment]
Can you try changing to direct x?

**Comment by mysteGH** [comment]
> Can you try changing to direct x?

Similar issue once switching but nowhere near as harsh, I managed to replay the moment some artefacts appeared with ctrl+f11 screen open and got some interesting documentation of the issue, cutting down the clip now.

**Comment by mysteGH** [comment]
https://github.com/ppy/osu/assets/96603029/cd83f08e-ce77-4be2-9241-7d4f03f98e5b

at the 5 second mark my monitor turns off very briefly and then artefacts. curious issue o.O

**Comment by peppy** [comment]
Frame graphs are looking perfect, so this must be a frame pacing issue. Something between the game and your screen. You could start by forcing vsync on and seeing if the game feels good again. Or if your monitor supports it, gsync.
